G/GibbsSeaWater/build_tarballs.jl

```diff
@@ -35,4 +34,4 @@ dependencies = [
 ]
 
 # Build the tarballs, and possibly a `build.jl` as well.
-build_tarballs(ARGS, name, version, sources, script, platforms, products, dependencies)
+build_tarballs(ARGS, name, version, sources, script, platforms, products, dependencies; julia_compat="1.7")
```

Comment:
Any specific reasons for requiring julia v1.7 and not 1.6?

Also, note that changing the julia compat bound means that you ***must*** change the version number, see #2763